Home brewed.
This is to fix info syntax for two e-mail addresses.
Will be sent up-stream.

--- screen-4.6.1/doc/screen.texinfo.orig	2017-07-28 04:44:52.340994806 -0700
+++ screen-4.6.1/doc/screen.texinfo	2017-07-28 04:45:22.327570761 -0700
@@ -5796,8 +5796,8 @@
 ============
 
 @verbatim
-     Thomas Renninger <treen@suse.com>,
-     Axel Beckert <abe@deuxchevaux.org>,
+     Thomas Renninger <treen@@suse.com>,
+     Axel Beckert <abe@@deuxchevaux.org>,
      Ken Beal <kbeal@@amber.ssd.csd.harris.com>,
      Rudolf Koenig <rfkoenig@@informatik.uni-erlangen.de>,
      Toerless Eckert <eckert@@informatik.uni-erlangen.de>,
